dockstore-webservice/src/main/java/io/dockstore/webservice/core/WorkflowVersion.java

```diff
@@ -66,12 +72,14 @@ public String getWorkingDirectory() {
     public void updateByUser(final WorkflowVersion workflowVersion) {
         super.updateByUser(workflowVersion);
         workflowPath = workflowVersion.workflowPath;
+        lastModified = workflowVersion.lastModified;
```

Comment:
Not clear why you need to do this. Not saying you don't, just don't understand.

Reply:
Was following that pattern that was in place here and in Tag.java of adding the new fields in the update functions.